spec/integration/zanox_integration_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'support/capybara_app_helper'
+
+RSpec.describe "Zanox Integration" do
+
+  before do
+    setup_app(action: :zanox) do |tracker|
+    tracker.handler(:zanox, { account_id: '12345H123456789' } )
+  end
```

Comment:
never seen syntax errors because of indentation ;)

the `end` on L8 is the closing from the block on L6, which should be indented, directly with L7.
the `end` on L11 is the closing from the before block on L5.